ParserXML/Parser.cs

```diff
@@ -1,46 +1,74 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
+using System.Runtime.Remoting.Services;
 using System.Xml.Linq;
 
-public class Contact
-{
-    public string Name { get; set; }
-    public Dictionary<string, string> Phones { get; set; }
-    public Address Address { get; set; }
-    public decimal? NetWorth { get; set; }
-
-    public Contact()
-    {
-        Phones = new Dictionary<string, string>();
-    }
-}
-
-public class Address
-{
-    public string Street1 { get; set; }
-    public string City { get; set; }
-    public string State { get; set; }
-    public string Postal { get; set; }
-}
-
 public class Program
 {
+    /// <summary>
+    /// New list
```

Comment:
In summary must be some usfeull info, single note that it is list didn't provide any information.